tensorboard/plugins/custom_scalar/README.md

```diff
@@ -73,61 +73,10 @@ to change the layout over time.
 
 ### Example Code
 
-```python
-from tensorboard import summary as summary_lib
-from tensorboard.plugins.custom_scalar import layout_pb2
-import tensorflow as tf
-
-step = tf.placeholder(tf.float32, shape=[])
-
-with tf.name_scope('loss'):
-  # Specify 2 different loss values, each tagged differently.
-  summary_lib.scalar('foo', tf.pow(0.9, step))
-  summary_lib.scalar('bar', tf.pow(0.85, step + 2))
-
-# Log metric baz as well as upper and lower bounds for making a margin chart.
-middle_baz_value = step + 4 * tf.random_uniform() - 2
-summary_lib.scalar('baz', middle_baz_value)
-summary_lib.scalar('baz_lower', middle_baz_value - 0.3 - tf.random_uniform())
-summary_lib.scalar('baz_upper', middle_baz_value + 0.3 + tf.random_uniform())
-
-with tf.name_scope('trigFunctions'):
-  summary_lib.scalar('cosine', tf.cos(step))
-  summary_lib.scalar('sine', tf.sin(step))
-  summary_lib.scalar('tangent', tf.tan(step))
-
-merged_summary = tf.summary.merge_all()
-
-with tf.Session() as sess, tf.summary.FileWriter('/tmp/logdir') as writer:
-  # We only need to specify the layout once (instead of per step).
-  summary_lib.custom_scalar_pb(layout_pb2.Layout(
-    category=[
-      layout_pb2.Category(
-        title='losses in 1 chart',
-        chart=[
-            layout_pb2.Chart(
-                title='losses',
-                tag=[r'loss.*']),
-        ]),
-      layout_pb2.Category(
-        title='trig functions',
-        chart=[
-            layout_pb2.Chart(
-                title='wave trig functions',
-                tag=[r'trigFunctions/cosine', r'trigFunctions/sine']),
-            # The range of tangent is different. Lets give it its own chart.
-            layout_pb2.Chart(
-                title='tan',
-                tag=[r'trigFunctions/tangent']),
-        ],
-        # This category we care less about. Lets make it initially closed.
-        closed=True),
-    ]))
-
-  for i in xrange(42):
-    summary = sess.run(merged_summary, feed_dict={step: i})
-    writer.add_summary(summary, global_step=i)
-```
+See [custom_scalar_demo.py](custom_scalar_demo.py) for an example of collecting
```

Comment:
(Grr at GitHub for not letting me comment on the appropriate line.)

Line 83 in the new file says "The above logic produces this custom scalar dashboard." However, now that we're just referencing the demo.py file, "above logic" isn't quite right.  Also, the demo.py itself doesn't produce that exact dashboard shown in the screenshot since it now shows a margin plot for losses.

I think we probably just want to change the wording to say "Example of the custom scalar dashboard UI" or something generic?

Reply:
Thanks for catching the outdated comments! We now use that title and show an updated screenshot.